source/docs/index.md

```diff
@@ -1,102 +1,22 @@
-title: Documentation
+title: Getting Started
 ---
-Welcome to the Embark documentation. If you encounter any problems when using Embark, have a look at the  [troubleshooting guide](troubleshooting.html), raise an issue on [GitHub](https://github.com/embark-framework/embark/issues) or ask in the [Gitter Chatroom](https://gitter.im/embark-framework/Lobby).
+Welcome to the Embark documentation! The following guides will help you guiding your way through your journey of building decentralized applications. If you encounter any problems when using Embark, or find anything unclear described here, please have a look at the [troubleshooting guide](troubleshooting.html), raise an issue on [GitHub](https://github.com/embark-framework/embark/issues) or ask in the [Gitter Chatroom](https://gitter.im/embark-framework/Lobby) respectively.
 
 ## What is Embark?
 
-Embark is a fast, simple and powerful framework to develop and deploy Decentralized Applications (DApps). 
+Embark is a fast, easy to use, and powerful developer environment to build and deploy decentralized applications, also known as "DApps". It integrates with [Ethereum](https://ethereum.org/) blockchains, decentralized storages like [IPFS](https://ipfs.io) and [Swarm](https://swarm-guide.readthedocs.io/en/latest/introduction.html), and decentralized communication platforms like [Whisper](https://github.com/ethereum/wiki/wiki/Whisper) and [Orbit](https://github.com/orbitdb/orbit).
```

Comment:
Should probably remove Orbit for now, as we don't support it right now

Reply:
Done :)